docs/serializers.md

```diff
@@ -810,6 +811,60 @@ fun main() {
 
 <!--- TEST --> 
 
+### Specifying serializer globally using typealias
+
+kotlinx.serialization tends to be an always-explicit framework when it comes to serialization strategies: normally,
+they should be explicitly mentioned in `@Serialiazble` annotation. Therefore, we do not provide any kind of global serializer
+configuration (except for [context serializer](#contextual-serialization) mentioned later).
+
+However, in projects with big amount of files and classes in them it may be inconvenient to specify `@file:UseSerializers`
+every time, especially for classes like `Date` or `Instant` that have fixed strategy of serialization across the project.
+For such cases, it is possible to specify serializers using `typealias`es, as they preserve annotations, including serialization ones:
+<!--- INCLUDE
+import java.util.Date
+import java.text.SimpleDateFormat
+  
+object DateAsLongSerializer : KSerializer<Date> {
+    override val descriptor: SerialDescriptor = PrimitiveSerialDescriptor("Date", PrimitiveKind.LONG)
```

Comment:
We tend to recommend using `serialName` as a unique identifier. Here we do have two serializer with the same identifier

Reply:
It's not visible to user btw, but OK